.travis.yml

```diff
@@ -1,28 +1,38 @@
 language: scala
-scala:
-- 2.10.6
-jdk:
-- oraclejdk8
+
+matrix:
+  include:
+    - scala: 2.12.3
+      jdk: oraclejdk8
+      env:
+        - SBT_VERSION=1.0.1
+    - scala: 2.10.6
```

Comment:
This might need to be `2.12.3` because sbt version in build.properties is `1.0.1`, but we are compiling for 0.13.16 which requires `2.10.6`. It might be a trial/error to see if one is faster than the other.